guides/source/development_dependencies_install.md

```diff
@@ -376,3 +376,11 @@ command inside of the `activestorage` directory to install the dependencies:
 ```bash
 yarn install
 ```
+
+WARNING: Extracting previews, tested in ActiveStorage's test suite requires
+third-party applications, `ffmpeg` for video and `mutool` for PDFs. Tests
+will raise errors with these applications installed. These libraries are
```

Comment:
I guess you meant `without` rather than `with`?

Reply:
Yes, thank you.
Corrected and tweaked lines a bit.